src/components/tables/DataTable.vue

```diff
@@ -116,7 +116,7 @@ export default {
       }
 
       if (dataCell.type && dataCell.type === 'dynamic') {
-        return parseFloat($(node).find('p').text().replace(/,/g, ''));
+        return $(node).find('p').data('value');
```

Comment:
Are you able to add the `data-value` attribute to the `node` instead of to `p`?

This way we can use:

```suggestion
        return node.dataset.value;
```

...without needing to use jQuery for querying the DOM. It's more efficient for larger tables.